### PR TITLE
Add readthedocs yml for pip installation

### PR DESCRIPTION
**Pull request by agoscinski**
The commit message and the discussion in #76 should explain the purpose of this branch sufficient.

### CODE REVIEW

**Comment by agoscinski** [comment]
Here the built of readthedocs for this branch, the notebook can be run without problems
https://scikit-cosmo.readthedocs.io/en/feat-add_readthedocs_yml/read-only-examples/PlotLFRE.html

File: .readthedocs.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - pdf
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+  version: 3.7
+  install:
+    - requirements: docs/requirements.txt
+    - method: pip
+      path: .
+  system_packages: true
```
Comment (Luthaf) [inline .readthedocs.yaml:23]: Why set this one to true? I'm afraid this might hide weird bugs later by picking a different version of some dependency

Comment (agoscinski) [reply .readthedocs.yaml:23]: Thats true, it was in the copy of the yml file, I'll remove it

Comment (agoscinski) [reply .readthedocs.yaml]: by removing it it switches to the default behaviour which is false

**Review by rosecers — APPROVED** [review]
Hey Alex, this looks good! Thanks for solving this issue!

**Comment by agoscinski** [comment]
So I removed the download as html and epub option which you had in the readthedocs (you have to click on the lower left corner), if someone wants to add it, one has to just add epub and localmedia to the formats in the yml.

**Pull request merged**